src/main/java/net/mcreator/ui/procedure/ProcedureSelector.java

```diff
@@ -79,6 +79,12 @@ public ProcedureSelector(@Nullable IHelpContext helpContext, MCreator mcreator,
 
 	public ProcedureSelector(@Nullable IHelpContext helpContext, MCreator mcreator, String eventName, Side side,
 			boolean allowInlineEditor, @Nullable VariableType returnType, Dependency... providedDependencies) {
+		this(helpContext, mcreator, eventName, eventName, side, allowInlineEditor, returnType, providedDependencies);
+	}
+
+	public ProcedureSelector(@Nullable IHelpContext helpContext, MCreator mcreator, String eventName,
```

Comment:
Add JavaDOC on what is procedure and event name

Reply:
✔️ Fixed